aggregator/README.md

```diff
@@ -133,7 +133,7 @@ Tests are defined in `test`. Running them directly is a bit verbose because of
 the deno flags you need:
 
 ```sh
-deno test -j --allow-net --allow-env --allow-read --unstable
```

Comment:
I ran into issues running the tests in parallel, it appears that when the Bundle/BundleSubmit/Eth service tests are running in parallel they can submit a duplicate L1/L2 nonce for the `VerificationGateway.processBundle` call. This may need more triage outside of this work.

Reply:
The above is noted in the `premerge` script/program, so the update above matches what is expected.